resources/recipes.py

```diff
@@ -676,7 +676,7 @@ def item_tag(namespace: str, _variant: str) -> str:
 
         # Tools
         if 'tool' in metal_data.types:
-            anvil_recipe(rm, '%s_tuyere' % metal, '#forge:double_sheet/%s' % metal, 'tfc:metal/tuyere/%s' % metal, metal_data.tier, Rules.bend_last, Rules.bend_second_last)
+            anvil_recipe(rm, '%s_tuyere' % metal, item_tag('forge', 'double_sheet'), 'tfc:metal/tuyere/%s' % metal, metal_data.tier, Rules.bend_last, Rules.bend_second_last)
```

Comment:
Could have fixed the item result to match everything else using `item('tuyere)` but not a huge deal.

Just something to keep in mind when working in the future that it's best to update stuff in one shot while your already there vs just leaving it, Keeps things cleaner for the future. There's also likely other places this could apply too.